test/unit/sql/data-types.test.js

```diff
@@ -282,6 +282,33 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
       });
     });
 
+    suite('GEOMETRY', function () {
+      testsql('GEOMETRY', DataTypes.GEOMETRY, {
+        default: 'GEOMETRY',
+        postgres: 'GEOMETRY'
+      });
+	
+      testsql('GEOMETRY(\'POINT\')', DataTypes.GEOMETRY('POINT'), {
+        default: 'GEOMETRY(POINT)',
+        postgres: 'GEOMETRY(POINT)'
+      });
+
+      testsql('GEOMETRY(\'LINESTRING\')', DataTypes.GEOMETRY('LINESTRING'), {
+        default: 'GEOMETRY(LINESTRING)',
+        postgres: 'GEOMETRY(LINESTRING)',
+      });
+
+      testsql('GEOMETRY(\'POLYGON\')', DataTypes.GEOMETRY('POLYGON'), {
+        default: 'GEOMETRY(POLYGON)',
+        postgres: 'GEOMETRY(POLYGON)'
+      });
+
+      testsql('GEOMETRY(\'POINT\',4326)', DataTypes.GEOMETRY('POINT', 4326), {
+        default: 'GEOMETRY(POINT,4326)',
```

Comment:
I believe this should be just `POINT` in the general case (for [mysql](https://dev.mysql.com/doc/refman/5.0/en/spatial-datatypes.html) at least